#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_fix_libX11_linkage.dpatch by Christopher James Halse Rogers <raof@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Explicitly link to libX11 to resolve missing symbols
## DP: on link.

@DPATCH@
Index: gnome-do/libdo/Makefile.am
===================================================================
--- gnome-do.orig/libdo/Makefile.am	2012-01-19 03:34:08.447686546 +0800
+++ gnome-do/libdo/Makefile.am	2012-01-19 03:51:22.161371463 +0800
@@ -16,7 +16,7 @@
 	eggaccelerators.c
 
 libdo_la_LDFLAGS = -export-dynamic -module -avoid-version
-libdo_la_LIBADD = $(LIBDO_LIBS)
+libdo_la_LIBADD = $(LIBDO_LIBS) -lX11
 
 maintainer-clean-local:
 	rm -f Makefile.in
